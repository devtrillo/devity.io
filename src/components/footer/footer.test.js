import { render } from '@testing-library/react';
import React from 'react';
import Footer from './index';

describe('The footer component', function () {
  it('should not change', function () {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
