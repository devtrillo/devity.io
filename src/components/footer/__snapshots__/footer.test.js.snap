// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The footer component should not change 1`] = `
<div>
  <footer
    class="footer"
  >
    Footer
  </footer>
</div>
`;
